oci-seccomp-bpf-hook.go

```diff
@@ -354,8 +387,48 @@ func generateProfile(c map[string]int, fileName string) error {
 	return nil
 }
 
+func parseAnnotation(annotation string) (outputFile string, inputFile string, err error) {
+	annotationSplit := strings.Split(annotation, ";")
+	if len(annotationSplit) > 2 {
+		return "", "", fmt.Errorf("The annotation must have only one \";\"")
+	}
+	for _, paths := range annotationSplit {
+		if strings.HasPrefix(paths, "if:") {
+			inputFile = strings.TrimSpace(strings.TrimPrefix(paths, "if:"))
+			if !filepath.IsAbs(inputFile) {
+				return "", "", fmt.Errorf("The path to the input file is not absolute")
+			}
```

Comment:
We need to check if the input file exists.

Reply:
... and if we can actually parse it. It would be a bad user experience if start tracing and only notice at the every end that the input file is malformed.

Reply:
Let's print the path as well: `..absolute: %q", path)`